plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestClickHouseConnectorTest.java

```diff
@@ -58,4 +60,11 @@ public void testCreateTableAsSelectWithTableComment()
         assertThatThrownBy(super::testCreateTableAsSelectWithTableComment)
                 .hasMessageMatching("(?s).* Syntax error: .* COMMENT 'test comment'.*");
     }
+
+    @Override
+    protected OptionalInt maxTableNameLength()
+    {
+        // The numeric value depends on file system
+        return OptionalInt.of(255 - ".sql.tmp".length());
```

Comment:
where comes `.sql.tmp` from?

Reply:
It comes from ClickHouse internal implementation.